addons/note/static/src/scss/note.scss

```diff
@@ -3,13 +3,18 @@
     text-decoration: line-through;
 }
 
-.o_note_form_view.o_form_view {
+.o_note_form_view {
     .o_form_sheet_bg {
         display: flex;
         flex-direction: column;
         padding: 0;
+        width: 100%;
+        max-width: 1200px;
         .o_form_statusbar {
             margin: 0;
+            .o_statusbar_buttons:empty {
+                display: none !important;
+            }
```

Comment:
This looks like a hack. Why is it needed now while it wasn't before ? It seems to me something might be off outside of the context of Note.

Reply:
The HTML structure was different before.
There are three elements in the status bar :
1. `.o_statusbar_buttons` (empty)
2. `[name=tag_ids]`
3. `.o_statusbar_status`

Before, they were put next to each other, and `.o_statusbar_status` had a `margin-left: auto`. The empty `o_statusbar_buttons` had no width so `[name=tag_ids]` was aligned to the left.

Now, it uses flex boxes so the fact that `.o_statbar_buttons` has no width doesn't matter, it still occupies a column", and `[name=tag_ids]` ends up in the middle. This is what I fixed with this CSS. I figured it might be best to do this here because I don't know if that's what we want in _every_ module and that's pretty hard to find out.